lib/mayaUsd/render/mayaToHydra/renderOverride.cpp

```diff
@@ -287,6 +287,7 @@ MtohRenderOverride::~MtohRenderOverride()
         MMessage::removeCallbacks(panelAndCallbacks.second);
     }
 
+    if(!_allInstances.empty())
     {
         std::lock_guard<std::mutex> lock(_allInstancesMutex);
         _allInstances.erase(
```

Comment:
Trying to erase an empty range was asserting in Debug.

Reply:
Great catch.